test/specs/TC6_Footer_links.js

```diff
@@ -0,0 +1,83 @@
+
+import loginPage from '../pageobjects/login.page.js'
+import inventoryPage from '../pageobjects/inventory.page.js'
+import { config } from '../../config.js';
+const { username, password } = config.valid_user;
+
+describe('TC6 - Footer link test', () => {
+    before(async () => {
+        await loginPage.open()
+        await loginPage.login(username, password)
+    });
+
+    it('should check that Twitter opens in new tab', async () => {
+        const isInventoryPageOpen = await inventoryPage.isOpen()
+        await expect(isInventoryPageOpen).toBe(true)
+        const originalWindowHandle = await browser.getWindowHandle();
+        await inventoryPage.clickTwitter();
+        await browser.waitUntil(
+            async () => (await browser.getWindowHandles()).length === 2,
+            {
+                timeout: 5000,
+                timeoutMsg: 'Expected a new tab to open after clicking Twitter icon'
+            }
+        );
+
+        const windowHandles = await browser.getWindowHandles();
+        const newWindowHandle = windowHandles.find(handle => handle !== originalWindowHandle);
+        await browser.switchToWindow(newWindowHandle);
+        const url = await browser.getUrl();
+        expect(url).toContain('x.com/saucelabs');
+        await browser.closeWindow();
```

Comment:
@karenn32 'closeWindow' and 'switchToWindow' must be added to the 'afterEach' block, since these are the same for each 'it' block.